.github/workflows/version.yml

```diff
@@ -50,8 +51,7 @@ jobs:
         git config --local user.email "test@github.com"
```

Comment:
Do you use "test@github.com" to indicate that a GH-runner initiated this change? If I understand correctly, there is no message, but I assume that the ledger will still associate the commit with a user?

Reply:
Yea I don't really know if its needed, especially now as I'm amending the commit.